src/sinks/kafka/request_builder.rs

```diff
@@ -88,7 +88,10 @@ fn get_headers(event: &Event, headers_key: &Option<String>) -> Option<OwnedHeade
                         let mut owned_headers = OwnedHeaders::new_with_capacity(headers_map.len());
                         for (key, value) in headers_map {
                             if let Value::Bytes(value_bytes) = value {
-                                owned_headers = owned_headers.add(key, value_bytes.as_ref());
+                                owned_headers = owned_headers.insert(Header {
+                                    key,
+                                    value: Some(value_bytes.as_ref()),
+                                });
```

Comment:
What is the difference here? Or was it just an API change to the crate?

Reply:
It's an API change in the upstream crate.